workflows/genomes_generation.nf

```diff
@@ -101,10 +101,10 @@ workflow GGP {
     // ---- combine data for reads and contigs pre-processing ---- //
     groupReads = { meta, assembly, fq1, fq2 ->
         if (fq2 == []) {
-            return tuple(meta, assembly, [fq1])
+            return tuple(meta + [single_end: true], assembly, [fq1])
```

Comment:
you probably don't need to do `+ [single_end: true]` if your samplesheet has the `single_end` column, because then your `schema_input.json` can put the value of it inside meta automatically like here: https://github.com/EBI-Metagenomics/asv-gen/blob/dev/assets/schema_input.json

Reply:
I had a look at your changes and I've resolved all the other ones thanks, this is the only one left from my comments before I approve it 😄

Reply:
I saw that comment, I will not modify that code **yet** because we have already input samplesheets without `single_end` column. I need to launch pipeline with them. I guess having that meta modification is not too bad. I will fix support of additional column after processing all marine studies (~4 left) not to be in rush. Thanks!